src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -316,12 +322,18 @@ public static class TextCatalog
                 outputColumnName, inputColumnName, ngramLength, skipLength, useAllLengths, maximumNgramsCount, weighting);
 
         /// <summary>
-        /// Produces a bag of counts of ngrams (sequences of consecutive words) in <paramref name="inputColumnNames"/>
-        /// and outputs bag of word vector as <paramref name="outputColumnName"/>
+        /// Create a <see cref="WordHashBagEstimator"/>, which takes the data from the multiple columns specified in <paramref name="inputColumnNames"/>
+        /// to a new column: <paramref name="outputColumnName"/> and produces a vector of counts of hashed n-grams.
         /// </summary>
-        /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.</param>
-        /// <param name="inputColumnNames">Name of the columns to transform.</param>
+        /// <remarks>
+        /// <see cref="WordHashBagEstimator"/> is different from <see cref="NgramHashingEstimator"/> in a way that <see cref="WordHashBagEstimator"/>
+        /// tokenizes text internally while <see cref="NgramHashingEstimator"/> takes tokenized text as input.
```

Comment:
in that the former .... and the latter ...
(we don't want to many similar links)